Riot/AppDelegate.m

```diff
@@ -1710,20 +1733,21 @@ - (void)addMatrixCallObserver
         [[NSNotificationCenter defaultCenter] removeObserver:matrixCallObserver];
     }
     
-    // Register call observer in order to handle new opened session
-    matrixCallObserver = [[NSNotificationCenter defaultCenter] addObserverForName:kMXCallManagerNewCall object:nil queue:[NSOperationQueue mainQueue] usingBlock:^(NSNotification *notif)
-    {
-        
+    void (^handleNotification)(NSNotification *) = ^(NSNotification *notif) {
```

Comment:
Why do you define a block and move away the code from where it is called?

From maintenance point of view and code readability, inline block code is easier. It makes asynchronous things look like sequential. Before GCD, there were delegate callbacks everywhere. It was a pain to follow what is happening on code you did not write or wrote but too long time ago for your memory.

There can be good reason for defining a block like the same block code called several time. But here, I do not see the reason.